packages/components/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "7.7.0",
+  "version": "7.7.1",
```

Comment:
I think it's technically not a bugfix, so one could argue that this could be 7.8.0 Version bump? No strong opinion from my side tho

Reply:
will change to 7.8.0 -> good point 🙏